medusa/backup_manager.py

```diff
@@ -110,8 +110,9 @@ def register_backup(backup_name, is_async):
 
             if backup_name in BackupMan.__instance.__backups:
                 logging.debug("Registered backup name {} found existing, replacing with new".format(backup_name))
```

Comment:
**Issue:** This debug logging is inaccurate. We're only replacing with new if `overwrite_existing` is `True`, which is only checked on the next line.

Reply:
Agreed. I've simply removed the line.